solution/ui/regulations/css/scss/partials/_search.scss

```diff
@@ -164,8 +164,7 @@
 
     input {
         width: 100%;
-        border: 1px solid $border_color;
-        border-right: none;
+        border: none;
```

Comment:
I made the change here because this was messing with the vuetify component.  I also checked and this isnt actually being consumed by the original version.  The headers css actually overwrites the search.scss file for the search box.    Vuetify has weird stuff going on when you have a global css file and since the search css is so specific I couldnt get it to overwrite unless I removed scope for some reason.  This fixes this without impacting anything else.